UefiPayloadPkg/Library/FdtParserLib/FdtParserLib.c

```diff
@@ -282,6 +288,15 @@ ParseReservedMemory (
           PlatformAcpiTable->Header.Revision = UNIVERSAL_PAYLOAD_ACPI_TABLE_REVISION;
           PlatformAcpiTable->Header.Length   = sizeof (UNIVERSAL_PAYLOAD_ACPI_TABLE);
         }
+      } else if (AsciiStrnCmp (TempStr, "smbios", AsciiStrLen ("smbios")) == 0) {
+        DEBUG ((DEBUG_INFO, " build smbios, NumberOfBytes:%x", NumberOfBytes));
+        BuildMemoryAllocationHob (StartAddress, NumberOfBytes, EfiReservedMemoryType);
+        SmbiosTable = BuildGuidHob (&gUniversalPayloadSmbios3TableGuid, sizeof (UNIVERSAL_PAYLOAD_SMBIOS_TABLE) + sizeof (SMBIOS_TABLE_3_0_ENTRY_POINT));

```

Comment:
gUniversalPayloadSmbios3TableGuid HOB only a pointer there, no need to add sizeof (SMBIOS_TABLE_3_0_ENTRY_POINT)

Reply:
Done.

Reply:
resolved